WMCDuplicateRemover.Tests/Wrappers/EventLogEntryWrapperTests.cs

```diff
@@ -1,19 +1,12 @@
 ﻿using NUnit.Framework;
-using System;
 
 namespace WMCDuplicateRemover.Tests.Wrappers
 {
     [TestFixture]
     public class EventLogEntryWrapperTests
     {
-        [TestFixtureSetUp]
-        public void SetUp()
-        {
-
-        }
-
```

Comment:
So am I to assume that one point you included the setup and teardown methods for unit testing even though you didn't use them?  Just like I did?